src/core/Expander/ExpanderGroup/ExpanderGroup.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv, HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
+import { baseStyles } from './ExpanderGroup.baseStyles';
+import { VisuallyHidden } from '../../../components';
+
+const baseClassName = 'fi-expander-group';
+const openClassName = `${baseClassName}--open`;
+const expandersContainerClassName = `${baseClassName}_expanders`;
+const openAllButtonClassName = `${baseClassName}_all-button`;
+
+type ToggleAllExpanderState = {
+  toState: boolean;
```

Comment:
I think this might be clearer with a more descriptive name. Now it's named kinda like a function even though it isn't one. Could it be "allOpen" or something along those lines?

Reply:
This is the original name and indicates the target state that the expanders should switch to unless already there. Maybe we could use `targetState` or `targetOpenState`? Any thoughts @ketsappi ?

Reply:
Ah, remembered the logic a bit differently. Still I'd maybe rather have the the name tell what state the value represents and handle what to do with it elsewhere. It feels likea a double edged sword in this case though. I'd be down with `targetState` or `targetOpenState` as well, since they are quite discriptive already.

Reply:
That would be ok, since it does not represent the current state nor necessarily the coming future state since some of the expanders might be controlled and refuse to change their state.

Reply:
This is no changed to targetOpenState along with the type name and other interface name changes.